app/helpers/workflow_helper.rb

```diff
@@ -4,7 +4,7 @@ def show_workflow_grid?
   end
 
   def render_workflow_grid
-    workflow_data = facet_tree('wf_wps')['wf_wps_ssim']
+    workflow_data = params[:all] ? facet_tree : facet_tree('wf_wps')['wf_wps_ssim']
```

Comment:
I'm a little confused what the expected behavior here is? Could you elaborate some more?